test/dotnet-new.Tests/CommonTemplatesTests.cs

```diff
@@ -12,6 +12,8 @@
 
 namespace Microsoft.DotNet.Cli.New.IntegrationTests
 {
+    // Temporary workaround for possible threading issues related to accessing VS COM objects.
```

Comment:
Might be good to link to an issue here. I was trying to create one, but made a post on First Responders about the problem I'm seeing with the issue generator. For reference, I tried to make this issue here, but it didn't work properly. https://github.com/dotnet/sdk/issues/43245

Reply:
The tracking issue is internal and not in github

Reply:
But I'm going to remove this since we solved this by using the older 1ES images in CI